Makefile

```diff
@@ -80,14 +80,17 @@ test: patch manifests generate fmt vet envtest ## Run tests.
 
 ##@ Build
 
+helm-plugins/cm-getter/cm-getter: $(shell find cmd/helm-cm-getter -type f -name '*.go')
+	go build -o $@ ./cmd/helm-cm-getter
```

Comment:
`./cmd/helm-cm-getter` here is the package that contains the source code for the `helm-plugins/cm-getter/cm-getter` binary.  
`$@` is the target, so `helm-plugins/cm-getter/cm-getter`.  
As for `-mod`:
```
	-mod mode
		module download mode to use: readonly, vendor, or mod.
		By default, if a vendor directory is present and the go version in go.mod
		is 1.14 or higher, the go command acts as if -mod=vendor were set.
		Otherwise, the go command acts as if -mod=readonly were set.
		See https://golang.org/ref/mod#build-commands for details.
```
Ultimately I think this command is correct. WDYT?

Reply:
You are right, my bad. Misread it.